packages/material-ui/src/Badge/Badge.js

```diff
@@ -95,7 +120,7 @@ Badge.propTypes = {
   /**
    * The content rendered within the badge.
    */
-  badgeContent: PropTypes.node.isRequired,
+  badgeContent: PropTypes.node,
```

Comment:
TS should be updated too if this prop is not mandatory any more.

Reply:
@kLabz Yes, it should. Do you want to take care of it? Well spotted.

Reply:
Done (#14186, I hope I was supposed to target `next`)